File: 100K Coding challenge/JAVA-100K/PositiveOrNegative.java
import java.util.Scanner;

public class PositiveOrNegative {
    public static void main (String ar[]) {
        Scanner sc = new Scanner(System.in);

        //Main code
        System.out.println("Enter a number:");
        int num = sc.nextInt();
        
        if (num<0) {
            System.out.println("Negative");
        } else if (num>0) {
            System.out.println("Positive");
        } else {
            System.out.println(0);
        }


        sc.close();
    }
    
}

File: C/Learn/7POSITIVE-OR-NEGATIVE/src/POSITIVE-OR-NEGATIVE.c
/*
 ============================================================================
 Name        : POSITIVE-OR-NEGATIVE.c
 Author      : 
 Version     :
 Copyright   : Your copyright notice
 Description : Hello World in C, Ansi-style
 ============================================================================
 */

#include <stdio.h>
#include <stdlib.h>

int main(void) {
	int num;
	setbuf(stdout, NULL);
	printf("Enter the number to be checked whether positive or negative : ");
	scanf("%d", &num);
	if (num<0) {
		printf("%d is negative", num);

	}
	else if (num>0){
		printf("%d is positive", num);
	}
	else {
		printf("%d is zero", num);
	}
	return EXIT_SUCCESS;
}
